Lib/fontTools/feaLib/ast.py

```diff
@@ -319,21 +319,20 @@ def addDefinition(self, definition):
         assert isinstance(definition, MarkClassDefinition)
         self.definitions.append(definition)
         for glyph in definition.glyphSet():
-            if glyph in self.definitions:
-                otherLoc = self.definitions[glyph].location
-                raise FeatureLibError(
-                    "Glyph %s already defined at %s:%d:%d" % (
-                        glyph, otherLoc[0], otherLoc[1], otherLoc[2]),
-                    definition.location)
+# Fixing this code fails tests!
```

Comment:
Each PR lives in its own branch on your fork.
I'm not sure commenting this code out is the right thing to do. I'd rather have the test cases fixed, so they don't declare markClass info for the same glyph more than once, or if they do assert that the correct exception is raised.
@brawer?